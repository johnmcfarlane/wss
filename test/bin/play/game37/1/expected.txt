 19  6,9  - HOW
 19  5,5  | WHIZ
 17  7,7  - WHO
 17  5,8  | ZHO
 16  5,6  | WIZ
 14  6,6  | WOE
 13  6,9  - HOI
 13  7,9  - IWI
 13  7,8  | THOW
 13  7,5  | WHIT
 13  8,9  - WHO
 12  8,7  - HOW
 12  8,9  - HOW
 12  6,7  - OW
 12  7,9  - OW
 12  6,9  - WO
 11  6,6  | HIE
 11  6,6  | HOE
 11  6,5  | HOWE
 10  6,7  - HI
 10  6,9  - HI
 10  7,9  - HI
 10  6,9  - HO
 10  6,5  | HOHE
 10  6,7  - OH
 10  6,9  - OH
 10  7,9  - OH
 10  8,9  - WO
  9  8,7  - HOI
  9  8,9  - HOI
  9  7,5  | HOIT
  9  8,5  | WHOA
  9  7,6  | WITH
  8  8,7  | HAW
  8  6,7  | HEII
  8  6,7  | HEW
  8  8,7  - HI
  8  8,9  - HI
  8  8,7  - HO
  8  8,9  - HO
  8  7,6  | HWT
  8  8,9  - IWI
  8  6,6  | OWE
  8  8,7  | WAH
  8  8,6  | WHA
  7  8,7  | HAH
  7  6,7  | HEH
  7  7,8  | THIO
  6  7,6  | HOTI
  6  8,6  | IWA
  6  8,5  | OHIA
  6  6,7  - OI
  6  6,9  - OI
  6  7,9  - OI
  6  7,6  | OWT
  6  7,8  | TOW
  6  7,8  | TWI
  6  7,8  | TWO
  6  6,7  | WEI
  6  7,6  | WIT
  6  8,6  | WOA
  6  7,6  | WOT
  5  8,8  | AHI
  5  8,8  | AHO
  5  8,8  | AW
  5  6,8  | EW
  5  8,7  | HAO
  5  6,7  | HEI
  5  8,6  | HIA
  5  7,6  | HIT
  5  7,6  | HOT
  5  7,8  | THO
  5  6,7  | WE
  5  7,7  | WT
  4  8,8  | AH
  4  6,8  | EH
  4  8,7  | HA
  4  6,7  | HE
  4  7,7  | HT
  4  8,9  - IO
  4  7,8  | TH
  3  8,7  | IAO
  3  7,8  | TOI
  2  8,8  | AI
  2  6,8  | EO
  2  7,7  | IT
  2  6,7  | OE
  2  7,8  | TI
  2  7,8  | TO
