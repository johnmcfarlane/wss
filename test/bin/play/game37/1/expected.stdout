 19  6,9  - HOW
 19  5,5  | WHIZ
 16  5,6  | WIZ
 14  6,6  | WOE
 13  6,9  - HOI
 13  7,9  - IWI
 13  7,5  | WHIT
 13  8,9  - WHO
 12  8,7  - HOW
 12  8,9  - HOW
 12  7,9  - OW
 12  6,9  - WO
 11  6,6  | HIE
 11  6,6  | HOE
 11  6,5  | HOWE
 10  6,7  - HI
 10  6,9  - HI
 10  6,9  - HO
 10  7,9  - OH
 10  8,9  - WO
  9  8,7  - HOI
  9  8,9  - HOI
  9  8,5  | WHOA
  9  7,6  | WITH
  8  8,7  | HAW
  8  6,7  | HEW
  8  8,7  - HI
  8  8,9  - HI
  8  8,7  - HO
  8  8,9  - HO
  8  8,9  - IWI
  8  6,6  | OWE
  8  8,6  | WHA
  7  8,7  | HAH
  7  6,7  | HEH
  7  7,8  | THIO
  6  8,5  | OHIA
  6  6,7  - OI
  6  6,9  - OI
  6  7,9  - OI
  6  7,8  | TOW
  6  7,8  | TWO
  6  7,6  | WIT
  6  7,6  | WOT
  5  8,8  | AHI
  5  8,8  | AW
  5  6,8  | EW
  5  8,7  | HAO
  5  7,6  | HIT
  5  7,6  | HOT
  5  7,8  | THO
  5  6,7  | WE
  4  8,8  | AH
  4  6,8  | EH
  4  8,7  | HA
  4  6,7  | HE
  2  8,8  | AI
  2  6,8  | EO
  2  7,7  | IT
  2  6,7  | OE
  2  7,8  | TI
  2  7,8  | TO
